jatsgenerator/generate.py

```diff
@@ -222,8 +222,15 @@ def set_article_datasets_header(self, parent):
 
     def set_data_availability(self, parent, poa_article):
         if poa_article.data_availability:
-            p_tag = SubElement(parent, "p")
-            p_tag.text = poa_article.data_availability
+            tag_name = 'p'
+            # Escape any unescaped ampersands
+            data_availability = etoolsutils.escape_ampersand(poa_article.data_availability)
+            # XML
+            tagged_string = '<' + tag_name + '>' + data_availability + '</' + tag_name + '>'
```

Comment:
As this is repeated a few times in this file, maybe move to a func?
```
def wrap_in_tags(tag_name, content):
    return '<{tag}>{content}</{tag}>'.format(tag=tag_name, content=content)
```

Reply:
I like it, used perhaps three times here, I will include it as a refactor in this PR, thanks @seanwiseman !